tests/script2.sh

```diff
@@ -0,0 +1 @@
+while true; do clear && pytest -s -n deskto_web; done
```

Comment:
`deskto_web` <--- is this a typo for `desktop_web`?

Reply:
Also, can we rename `script1.sh` to `mobile_native.sh` and rename `script2.sh` to `desktop_web.sh`?

Reply:
I was thinking that too, so I've updated the names of the scripts.

Yes typo. I had tested this from the compute engine but didn't check code into source control there, so that's why it worked there. Committing the code via my local machine. Updated.